### PR TITLE
As a Developer, I need to add installments for revolving credit loans…

### PR DESCRIPTION
**Pull request by florinparascan**
… via APIs

### CODE REVIEW

File: src/demo/DemoTestRepaymentService.java [hunk 1]
```diff
@@ -40,7 +42,7 @@ public static void main(String[] args) {
 			demoLoanAccount = DemoUtil.getDemoLoanAccount(testAccountId);
 			LOAN_ACCOUNT_ID = demoLoanAccount.getId();
 
-			repayemnts = testGetLoanAccountRepayments();
+			repayments = testGetLoanAccountRepayments();
```
Comment (mdanilkis) [inline src/demo/DemoTestRepaymentService.java]: Thanks for correcting this!  👍

File: src/demo/DemoTestRepaymentService.java [hunk 2]
```diff
@@ -138,6 +143,17 @@ public static void testUpdateLoanRepaymentsSchedule() throws MambuApiException {
 			}
 
 		}
+
```
Comment (mdanilkis) [inline src/demo/DemoTestRepaymentService.java]: Can we add comets with Mambu issue number here to clarify what scenario are we testing here and for future reference?

File: src/demo/DemoTestRepaymentService.java [hunk 3]
```diff
@@ -191,8 +208,8 @@ public static void testGetInvestorAccountRepayments() throws MambuApiException {
 		System.out.println("Total Repayments=" + repayemnts.size());
 		if (repayemnts.size() > 0) {
 			System.out.println("First Repayment  Due date=" + repayemnts.get(0).getDueDate().toString());
-			System.out.println("Last  Repayment  Due date="
-					+ repayemnts.get(repayemnts.size() - 1).getDueDate().toString());
+			System.out.println(
+					"Last  Repayment  Due date=" + repayemnts.get(repayemnts.size() - 1).getDueDate().toString());
 		}
```
Comment (mdanilkis) [inline src/demo/DemoTestRepaymentService.java]: Good test!

File: src/demo/DemoTestRepaymentService.java (hunk 2 above)
Comment (florinparascan) [inline src/demo/DemoTestRepaymentService.java]: Done.

**Pull request merged**